db_migration/alembic/versions/20211117_162939_7854bc67c695_simplify_drsobject.py

```diff
@@ -0,0 +1,60 @@
+"""simplify drsobject
+
+Revision ID: 7854bc67c695
+Revises: b28851e1a2ea
+Create Date: 2021-11-17 16:29:39.753168
+
+"""
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "7854bc67c695"
+down_revision = "b28851e1a2ea"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column("drs_objects", sa.Column("external_id", sa.String(), nullable=False))
+    op.add_column("drs_objects", sa.Column("md5_checksum", sa.String(), nullable=False))
+    op.add_column(
+        "drs_objects", sa.Column("registration_date", sa.DateTime(), nullable=False)
```

Comment:
ga4gh drs has a required "created_time" field for drs_objects, which I always assumed would be the creation time of the file on the researchers side. (https://ga4gh.github.io/data-repository-service-schemas/preview/release/drs-1.0.0/docs/)

Is this registration_date meant as a replacement for creation_time?

Reply:
I think the specs leave a lot time for interpretation:
```
Timestamp of content creation in RFC3339.
(This is the creation time of the underlying content, not of the JSON object.)
```
It's not clear whether they mean the creation time at the researcher side (that is something that we most likely cannot always get from the submitters) or the creation time in our system. The latter is roughly equivalent with the registration date/time to the DRS service. So I think we can use this definition as it is more convenient for us to implement.

Reply:
Would that be OK?

Reply:
Fine with me.